### ISSUE TITLE
Versionless features for Java / Jakarta EE Container Liberty features

### ISSUE THREAD

**Issue by jhanders34** [OpenLiberty/open-liberty#29571]
The information you provide here will be included in the Open Liberty GA release blog post ([example](https://openliberty.io/blog/2022/01/18/microprofile5-22001.html)), which will be published on [openliberty.io/blog/](https://www.openliberty.io/blog/), and potentially elsewhere, to promote this newly released feature/function of Open Liberty. For this post to be included in the GA issue please make sure that this is completed by the end of Friday following the GM (Wednesday). The beta and release blogs are created using automation and rely on you following the template's structure.  **DO NOT REMOVE/ALTER THE `<GHA>` TAGS THROUGHOUT THIS TEMPLATE.**

Please provide the following information:

1. If this was previously published in a beta blog post, then provide the link to that `OpenLiberty/open-liberty` beta blog post issue on the next line between the `<GHA-BLOG-BETA-LINK>` tags. If nothing has changed since the beta, you're done and can omit the remaining steps. If you need to make updates/alterations to the beta content, then do all the steps. 
   <GHA-BLOG-BETA-LINK></GHA-BLOG-BETA-LINK>

   <GHA-BLOG-SUMMARY>    
In 24.0.0.8, Open Liberty introduced versionless Java EE and Jakarta EE features.  Those new versionless features allow you to easily use features without needing to know what feature versions to use.

In 24.0.0.8, the versionless features did not include the `Container` features where the user can provide their own implementation of that Java EE or Jakarta EE component specification.  An example of such a feature is `facesContainer-4.0`.

In 24.0.0.9, Open Liberty added versionless features for the `Container` features.   These versionless features were added:

- `jpaContainer` / `persistenceContainer`
- `jsfContainer` / `facesContainer`
- `jsonbContainer`
- `jsonpContainer`

This `server.xml` configuration file uses Java EE platform `javaee-8.0` with versionless features `jpaContainer`, `jsfContainer`, `jsonbContainer`, and `jsonpContainer`:

```
    <!-- Enable features -->
    <featureManager>
        <platform>javaee-8.0</platform>
        <feature>jpaContainer</feature>
        <feature>jsfContainer</feature>
        <feature>jsonbContainer</feature>
        <feature>jsonpContainer</feature>
    </featureManager>
```

Learn more and check out the full collection of available platforms and versionless features in the link:{url-prefix}/docs/latest/reference/feature/versionless-features.html[Open Liberty docs]. Stay tuned for more versionless features and platforms in future releases.
   </GHA-BLOG-SUMMARY>

## What happens next?
- Add the label for the GA you're targeting: `target:YY00X`.
- Make sure this blog post is linked back to the Epic for this feature/function.
- Your paragraph will be included in the GA release blog post. It might be edited for style and consistency.
- You will be asked to review a draft before publication.
    - Once you've approved the code review, close this issue. 
- If you would _also_ like to write a standalone blog post about your update (highly recommended), raise an issue on the [Open Liberty blogs repo](https://github.com/OpenLiberty/blogs/issues/new/choose). State in the issue that the blog post relates to a specific release so that we can ensure it is published on an appropriate date (it won't be the same day as the GA blog post).

**Comment by cbridgha** [comment]
This all looks good

**Comment by hlhoots** [comment]
This looks good to me too.

**Comment by jhanders34** [comment]
Release blog was reviewed and released for 24.0.0.9, so closing the issue.
